### PR TITLE
autoencoder solution notebook and associated asset files

### PR DESCRIPTION
**Pull request by munnm**
Hi Benoit, can you give this autoencoder notebook a look? It's very similar to the gan one. But let me know if you don't have time and I can find someone else. 

(ps. i had originally made a branch called munn-autoencoder and tried to PR that but noticed a conflict with merging so abandoned that branch and moved it all to this clean branch munn-ae. sorry for any confusion)

### CODE REVIEW

File: courses/machine_learning/deepdive2/supplemental/solutions/autoencoder.ipynb [hunk 1]
```diff
@@ -0,0 +1,687 @@
+{
```
Comment (BenoitDherin) [inline courses/machine_learning/deepdive2/supplemental/solutions/autoencoder.ipynb]: <ul><li>GAN -&gt; autoencoder</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (BenoitDherin) [inline courses/machine_learning/deepdive2/supplemental/solutions/autoencoder.ipynb]: <ul><li>"Note, training GANs can be tricky" --&gt; ""</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

**Comment by BenoitDherin** [comment]
<p>LGTM. You can merge.</p>

File: courses/machine_learning/deepdive2/supplemental/solutions/autoencoder.ipynb (hunk 1 above)
Comment (munnm) [reply courses/machine_learning/deepdive2/supplemental/solutions/autoencoder.ipynb]: <p>fixed here and throughout. Thanks!</p>

**Pull request merged**